cdhweb/pages/templatetags/custom_tags.py

```diff
@@ -1,8 +1,18 @@
 import uuid
+
 from django import template
 
+from ..blocks.accordion_block import ProjectAccordion
+
 register = template.Library()
 
+
 @register.simple_tag(takes_context=True)
 def unique_id(context, prefix="id-"):
     return f"{prefix}{uuid.uuid4().hex}"
+
+
+@register.simple_tag
+def get_accordion_heading_display(choice_value):
```

Comment:
It is _wild_ to me that we have to do this, but it does seem to be the case :(

Reply:
So what's going on here? To clarify, we don't want to give the CMS user a choice (if that's what's happening). 
These accordions are basically hard-coded as the first content on the page (after the hero). There's no `h2` above them, therefore the heading level for these should be `h2`.
<img width="734" alt="Screenshot 2024-07-04 at 8 54 40 AM" src="https://github.com/springload/cdh-web/assets/1134713/3e009109-50d7-41af-b87f-f335b856b26b">

Reply:
I was thinking this was to do with heading _levels_, not the heading text. Ignore me, once again.